apps/bfd-model/bfd-model-codegen/src/main/java/gov/cms/bfd/model/codegen/RifLayoutsProcessor.java

```diff
@@ -224,7 +225,7 @@ private void process(PackageElement annotatedPackage) throws IOException {
                   "nameMiddleInitial")
               .setHeaderEntityAdditionalDatabaseFields(
                   createDetailsForAdditionalDatabaseFields(
-                      Arrays.asList("hicnUnhashed", "medicareBeneficiaryId")))
```

Comment:
@whytheplatypus Good call out, but `medicareBeneficiaryId` has been moved to the BeneficiariesHistory tab of `rif-layout-and-fhir-mapping.xlsx`. This is to enable loading of MBIs into the test data set, so tests could be written against the mbiHash. Any fields added via the `createDetailsForAdditionalDatabaseFields` were not testable as they did not utilize the RifLoader.

Reply:
Will the [`medicareBeneficiaryId` field on the `Beneficiaries`](https://github.com/CMSgov/beneficiary-fhir-data/blob/master/apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V10__Add_beneficiary_beneficiaryHistory_colmns.sql#L10) table still get populated?

Reply:
Yes, line 227 is referencing the `BeneficiariesHistory` table, which now includes `medicareBeneficiaryId` directly in its mapping file.

Since `medicareBeneficiaryId` is no longer referenced in any additionalDataField, line 1294 was dead code so it was removed.

Additionally, this PR adds test to assert the `BeneficiariesHistory` table populates the `medicareBeneficiaryId` column (this was previously not asserted): https://github.com/CMSgov/beneficiary-fhir-data/pull/180/files#diff-d7d0e43ba2ec6fc78d70d8a0fd0476a1R125

Thanks for having me double-check my logic! MBIs are so important, we want to make sure we don't mess up anything in our dataset.

Reply:
got it, ty for the explanation!